docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:jessie
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+ENV LANG C.UTF-8
+
+RUN apt-get update && apt-get install -y \
+    apt-utils
```

Comment:
Prevents warnings in subsequent installations - something about packages not being configured until apt-utils is installed (or the equivalent).